src/components/TimeQuerier/timeQuerier.test.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import moment from "moment";
+import { fireEvent } from "@testing-library/react";
+import TimeQuerier from "./";
+import { render } from "../../utils/customRender";
```

Comment:
shouldn't `render` be imported from testing-library?

Reply:
No we have a made a custom renderer remember that renders component along with the providers

Reply:
Makes sense